src/scene/container/RenderGroup.ts

```diff
@@ -200,18 +201,7 @@ export class RenderGroup implements Instruction
         childrenToUpdate.list[childrenToUpdate.index++] = child;
     }
 
-    // SHOULD THIS BE HERE?
-    public updateRenderable(container: Container)
```

Comment:
this seems like another public API being removed

Reply:
this one is kind of internal public iyswim? Not really meant to be used by others. Think its still worth depricating?